pkg/utils/validator/clusterresourceoverride.go

```diff
@@ -79,3 +92,28 @@ func validateClusterResourceOverrideResourceLimit(cro fleetv1alpha1.ClusterResou
 	}
 	return errors.NewAggregate(allErr)
 }
+
+// validateClusterResourceOverridePolicy checks if override rule is selecting resource by name.
+func validateClusterResourceOverridePolicy(cro fleetv1alpha1.ClusterResourceOverride) error {
+	allErr := make([]error, 0)
+	for _, rule := range cro.Spec.Policy.OverrideRules {
```

Comment:
isn't this the same as the resourceOverride? We should try to avoid code duplication